.eslintrc.js

```diff
@@ -6,38 +6,138 @@ module.exports = {
     env: {
         jest: true,
     },
-    settings: {
-        'import/resolver': {
-            node: {
-                extensions: ['.js', '.website.js', '.desktop.js', '.native.js', '.ios.js', '.android.js', '.config.js'],
-            },
-        },
-    },
     globals: {
         __DEV__: 'readonly',
     },
-    rules: {
-        'no-restricted-imports': [
-            'error',
-            {
-                paths: [
+    overrides: [
+        {
+            files: ['*.js', '*.jsx', '*.ts', '*.tsx'],
+            rules: {
+                'no-restricted-imports': [
+                    'error',
+                    {
+                        paths: [
+                            {
+                                name: 'react-native',
+                                importNames: ['useWindowDimensions'],
+                                message: 'Please use useWindowDimensions from src/hooks/useWindowDimensions instead.',
+                            },
+                            {
+                                name: 'react-native',
+                                importNames: ['TouchableOpacity', 'TouchableWithoutFeedback', 'TouchableNativeFeedback', 'TouchableHighlight'],
+                                message: 'Please use PressableWithFeedback and/or PressableWithoutFeedback from src/components/Pressable instead.',
+                            },
```

Comment:
I'm not sure if this PR is the right place to restrict these imports 🤔 Do you know why it wasn't added before?

Reply:
Not sure why it wasn't included before, but following these restrictions idea I think it's supposed to be there as well, because the developer can import the wrong component from this library too.

@hayata-suenaga Do you think we can include this restriction in this PR or leave it in a separate one?

Reply:
@fabioh8010 

ah I forgot RNGH exports touchable components. yes let's restrict them here with additional rule config! nice suggestion 👍

Reply:
cc @blazejkustra

Reply:
Done 👍